src/partials/head-prelude.hbs

```diff
@@ -2,7 +2,9 @@
     <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=5, minimum-scale=1">
     <meta name="msvalidate.01" content="B5163518CBE4A854B63801277FE5E35C" />
     <meta name="google-site-verification" content="ucqagxjVuq0lJZeLKs0F5AppzK111lNt3IoxU6mzlJE" />
+    {{#unless (page.attributes.no-index) }}
```

Comment:
:+1:

Reply:
Maybe we could be explicit and use the following when `page-no-index` is defined:

```hbs
<meta name="robots" content="noindex" />
```

Reply:
OK, I think I can make the change you have in mind.